### ISSUE TITLE
Training BatchNorm and Only BatchNorm: On the Expressive Power of Random Features in CNNs

### ISSUE THREAD

**Issue by icoxfog417** [arXivTimes/arXivTimes#1602]
## 一言でいうと

Batch Normalization(BN)の効果を検証した研究。正規化した値をスケーリング/シフトするパラメーター(γ/β)のみ学習し、他全てのネットワークのパラメーターを初期化時のまま固定し検証。BNのパラメーター学習のみでCIFAR-10で83%の精度を達成。γは1/4~1/2程度のチャンネルを抑制する働きが見られた。

### 論文リンク

https://arxiv.org/abs/2003.00152

### 著者/所属機関

Jonathan Frankle, David J. Schwab, Ari S. Morcos

* MIT CSAIL
* CUNY Graduate Center, Initiative for the Theoretical Sciences
* Facebook AI Research

### 投稿日付(yyyy/MM/dd)

2020/2/29
